gst/nnstreamer/nnstreamer_conf.c

```diff
@@ -44,10 +44,12 @@ typedef struct
   gchar *pathFILTERS[CONF_SOURCES];        /**< directory paths for FILTERS */
   gchar *pathDECODERS[CONF_SOURCES];       /**< directory paths for DECODERS */
   gchar *pathCUSTOM_FILTERS[CONF_SOURCES]; /**< directory paths for CUSTOM FILTERS */
+  gchar *valueTF_MEM_OPTMZ[CONF_SOURCES];  /**< value of TF_MEM_OPTMZ */
 
   gchar **filesFILTERS;        /**< Null terminated list of full filepaths for FILTERS */
   gchar **filesDECODERS;;      /**< Null terminated list of full filepaths for DECODERS */
   gchar **filesCUSTOM_FILTERS; /**< Null terminated list of full filepaths for CUSTOM FILTERS */
+  gchar *retTF_MEM_OPTMZ;
```

Comment:
Wouldn't this be better gboolean ? (it's true / false)

Reply:
@myungjoo 
Since the below method returns the value and it should handle various types, `gchar *` was taken as the return type and the type of the variable `retTF_MEM_OPTMZ`.
https://github.com/nnsuite/nnstreamer/blob/298fd44a8fb0a5fa39da30d10aaa769e6ac27d16/gst/nnstreamer/nnstreamer_conf.c#L357-L359